### DISCUSSION TITLE
How to scale active element

### DISCUSSION THREAD

**Discussion by MariuszRogowicz** [davidjerleke/embla-carousel#582] [Help] [answered]
When try to scale active element` &__slide` using `transform ` property, some of carousel items will thrown away.
This is only because we are overriding the normal transform3d property that embela uses. 

So how i can add scale, without breaks library work? 
I try to use just simple scale property, but this does not solve my problem. 
https://css-tricks.com/almanac/properties/s/scale/

![image](https://github.com/davidjerleke/embla-carousel/assets/86837961/86f6be88-c60b-4229-8f32-efd62c4e5d56)

**Comment by davidjerleke** [comment, marked as answer]
Hi @MariuszRogowicz,

You can use the approach in the [scale example](https://www.embla-carousel.com/examples/predefined/#scale) in the docs. It adds an additional element inside the slide element and scale that instead:

```html
<div class="embla">
  <div class="embla__container">

    <div class="embla__slide">
      <div class="embla__slide__scale" <!-- scale this element -->>
        Slide content here
      </div>
    </div>

    ...more slides
  </div>
</div>
``` 

**Note** that if you have loop enabled, scaling slides can break the carousel unless you follow the scale example approach in the docs. 

Best,
David
